lib/managers/ar_object_manager.dart

```diff
@@ -103,6 +109,28 @@ class ARObjectManager {
             onRotationEnd!(tappedNodeName, transform);
           }
           break;
+        case 'onScaleStart':
+          if (onScaleStart != null) {
+            final tappedNode = call.arguments as String;
+            onScaleStart!(tappedNode);
+          }
+          break;
+        case 'onScaleChange':
+          if (onScaleChange != null) {
+            final tappedNode = call.arguments as String;
+            onScaleChange!(tappedNode);
```

Comment:
does this actually update the scale in the Flutter representation of the AR node?